app/models/amusement_park.rb

```diff
@@ -1,3 +1,9 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :mechanics, through: :rides
+
+  def ride_exp
+    a = rides.select("rides.*, avg(mechanics.years_experience) as avg").joins(:mechanics).group("rides.id").order("avg desc")
+    # require 'pry'; binding.pry
+  end
 end
```

Comment:
I really like that you have added the relationship to mechanics on line 3. I'm curious if you can find a way to utilize that relationship in the `ride_exp` method.